src/CQRT11.jl

```diff
@@ -0,0 +1,28 @@
+#   Source: http://www.ai7.uni-bayreuth.de/test_problem_coll.pdf
+#   Problem 8 (p.31) in
+#   K. Schittkowski,
+#   "Test Examples for Nonlinear Programming Codes",
+#   Department of Computer Science, University of Bayreuth,
+#   D - 95440 Bayreuth, 2009.
+#
+# A. Montoison and L. Vincent, Montreal, 01/2018.
+
+export CQRT11
```

Comment:
This is not the problem name. This is the classification string (which should appear in the comments, as it is stated in the PDF). The problem name is HS8, but it is already present in this collection: https://github.com/JuliaSmoothOptimizers/OptimizationProblems.jl/blob/master/src/hs8.jl